include/nbl/builtin/hlsl/blit/parameters.hlsl

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_BLIT_PARAMETERS_INCLUDED_
+#define _NBL_BUILTIN_HLSL_BLIT_PARAMETERS_INCLUDED_
+
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace blit
+{
+
+struct parameters_t
+{
+	uint32_t3 dims; // input dimensions in lower 16 bits, output dimensions in higher 16 bits
```

Comment:
you can get rid of this hack, as we always compile with `-enable-16bit-types` and `-fvk-use-scalar-layout` on master